### PR TITLE
implement controller for ranking

### PR DESCRIPTION
**Pull request by bellatoris**

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #140** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/140" target="_blank">Graphite</a> 
    * **PR #141** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/141" target="_blank">Graphite</a> 
      * **PR #142** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/142" target="_blank">Graphite</a> 
        * **PR #143** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/143" target="_blank">Graphite</a> 
          * **PR #145** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/145" target="_blank">Graphite</a> 
            * **PR #146** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/146" target="_blank">Graphite</a> 
              * **PR #147** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/147" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/147?utm_source=stack-comment).

File: subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/in/controller/AccessibilityRankController.kt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    ): GetCountForNextRankPost200Response {
+        return GetCountForNextRankPost200Response(
+            countForNextRank = getCountForNextRankUseCase.handle(authentication.principal)
+        )
+    }
+
+    @PostMapping("/updateAccessibilityRanks")
+    fun getCountForNextRank() {
+        updateRanksUseCase.handle()
+    }
```
Comment (bellatoris) [inline subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/in/controller/AccessibilityRankController.kt:53]: 이거 IP로 막고 싶은게 spring security + 현재 스완이 구현해놓은 구조에서 손쉽게 가능한가여?

Comment (Zeniuus) [reply subprojects/bounded_context/accessibility/infra/src/main/kotlin/club/staircrusher/accessibility/infra/adapter/in/controller/AccessibilityRankController.kt:53]: IP...는 Spring Security로 안 해봐서 잘 모르겠는데, header에 담기는 거면 interceptor 추가로 쉽게 가능할 것 같아요.

**Review by Zeniuus — APPROVED** [review]

**Comment by bellatoris** [comment]
@bellatoris queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/147).

**Comment by bellatoris** [comment]
[Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/147) rebased this pull request as part of a merge.

**Comment by bellatoris** [comment]
@bellatoris merged this pull request with [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/147).

**Pull request merged**